dwave/optimization/src/nodes/mathematical.cpp

```diff
@@ -750,8 +1023,7 @@ template <>
 ReduceNode<std::plus<double>>::ReduceNode(ArrayNode* array_ptr) : ReduceNode(array_ptr, 0) {}
 
 template <class BinaryOp>
-ReduceNode<BinaryOp>::ReduceNode(ArrayNode* array_ptr)
-        : init(), array_ptr_(array_ptr) {
+ReduceNode<BinaryOp>::ReduceNode(ArrayNode* array_ptr) : init(), array_ptr_(array_ptr) {
```

Comment:
Please don't edit unrelated functions for formatting. It clutters the git history and without https://github.com/dwavesystems/dwave-optimization/issues/113 it's not even deterministic.